src/components/ComponentDemo/ComponentDemo.module.scss

```diff
@@ -172,6 +172,7 @@ fieldset.form-group {
 
 .editor-container pre,
 .editor-container textarea {
+  font-family: 'IBM Plex Mono' !important;
```

Comment:
Don't we have a token for the font family name? Or is that only in the theme? and do we need the important?